### ISSUE TITLE
Improve instructions: Nest an Anchor Element within a Paragraph

### ISSUE THREAD

**Issue by mtrivera** [freeCodeCamp/freeCodeCamp#16335]
#### Challenge Name
Basic HTML and HTML5: Nest an Anchor Element within a Paragraph
https://beta.freecodecamp.org/en/challenges/basic-html-and-html5/nest-an-anchor-element-within-a-paragraph

#### Issue Description
See #16289

**Comment by TheBGar** [comment]
Is this issue for the beta version still needing a fix?

**Comment by raisedadead** [comment]
@mtrivera yes, this is open for contribution please see linked issue for details, on what is needed.
